### PR TITLE
New page for pub bump

### LINKED ISSUE #6147: Add documentation for `dart pub bump`
Reference: https://github.com/dart-lang/pub/commit/e9ad2bcf945880146619700d66e66ddc3342c762

### PR DESCRIPTION
**Pull request by MaryaBelanger**
Fixes #6147 

Not sure if I wrote line 9 correctly: `$ dart pub bump <subcommand> [arguments]`

Let me know if any other info would be helpful to include!

### CODE REVIEW

File: src/content/tools/pub/cmd/pub-bump.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+description: Use dart pub bump to increase the version number of the current package.
+---
+
+_Bump_ is one of the commands of the [pub tool](/tools/pub/cmd).
+
+```plaintext
+$ dart pub bump <subcommand> [arguments]
+```
+
+This command increments the verion number of the current package.
```
Comment (sigurdm) [inline src/content/tools/pub/cmd/pub-bump.md:12]:
```suggestion
This command increments the version number of the current package.
```

File: src/content/tools/pub/cmd/pub-bump.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+It allows the subcommands `breaking`, `major`, `minor`,
+and `patch` to increment only a specific part of the version number.
```
Comment (sigurdm) [inline src/content/tools/pub/cmd/pub-bump.md:14]:
```suggestion
Use one of the subcommands `breaking`, `major`, `minor`,
or `patch` according to the type of increment desired. 

Refer to [semver](https://semver.org/spec/v2.0.0-rc.1.html) for guidance on versioning your package.
```

File: src/content/tools/pub/cmd/pub-bump.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+---
+title: dart pub bump
+description: Use dart pub bump to increase the version number of the current package.
+---
+
+_Bump_ is one of the commands of the [pub tool](/tools/pub/cmd).
+
+```plaintext
+$ dart pub bump <subcommand> [arguments]
```
Comment (sigurdm) [inline src/content/tools/pub/cmd/pub-bump.md:9]: The command currently doesn't have any arguments you can pass
```suggestion
$ dart pub bump <subcommand>
```

File: src/content/tools/pub/cmd/pub-bump.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+## Options
+
+For options that apply to all pub commands, see
+[Global options](/tools/pub/cmd#global-options).
+
+### `-n, --dry-run`
+
+Reports what would change for the version number, and shows the version diff,
+without changing anything.
```
Comment (sigurdm) [inline src/content/tools/pub/cmd/pub-bump.md:36]: This is not actually implemented (perhaps it should be though...)

**Review by sigurdm — APPROVED** [review]

File: src/content/tools/pub/cmd/pub-bump.md (hunk 4 above)
Comment (MaryaBelanger) [reply src/content/tools/pub/cmd/pub-bump.md:38]: Oh `dry-run` isn't implemented? [It comes up a few times in the PR though](https://github.com/dart-lang/pub/pull/4361/files#diff-f03751343a35e73516dca2f40772206134537c925f66d0505cc6c5c28e0b9976R25), is that just like making space for it in the future?

I'll remove it anyway for now to land the page, just curious!

Comment (sigurdm) [reply src/content/tools/pub/cmd/pub-bump.md]: Oh I'm sorry - you are totally right.

I was looking at `dart pub bump --help` but should of course look at `dart pub bump <subcommand> help`.

And it is indeed implemented (by me...).

Thanks for pointing this out!

Comment (MaryaBelanger) [reply src/content/tools/pub/cmd/pub-bump.md]: No worries! Thanks for explaining, I put it back and specified it's for each subcommand

**Pull request merged**